modules/javafx.controls/src/main/java/javafx/scene/control/TreeTableView.java

```diff
@@ -1848,6 +1856,16 @@ public void sort() {
         if (sortPolicy == null) return;
```

Comment:
Do we need to set sortingInProgress to false before returning from here?

Reply:
Yes, I think so. Good catch.

Reply:
Corrected in the next commit.